modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -909,6 +914,7 @@ func (h Handler) finalizeResponse(
 
 	// total duration spent proxying, including writing response body
 	repl.Set("http.reverse_proxy.upstream.duration", time.Since(start))
+	repl.Set("http.reverse_proxy.upstream.duration_sec", time.Since(start).Seconds()*1e3)
```

Comment:
Did this _sec sneak in here during testing? Either way, multiplying secs by 1000 doesn't seem accurate?

Reply:
It was too late last night for me to be making these changes, they're pretty wack right now.